sdk/identity/Azure.Identity/api/Azure.Identity.netstandard2.0.cs

```diff
@@ -129,4 +129,11 @@ public partial class UsernamePasswordCredential : Azure.Core.TokenCredential
         public override Azure.Core.AccessToken GetToken(Azure.Core.TokenRequestContext requestContext, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public override System.Threading.Tasks.ValueTask<Azure.Core.AccessToken> GetTokenAsync(Azure.Core.TokenRequestContext requestContext, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
     }
+    public partial class VisualStudioCredential : Azure.Core.TokenCredential
```

Comment:
Does it have to be public?

Reply:
Default constructor can be made protected. As for the credential itself, most of other credentials are public. @schaabs, what is our policy on that?

Reply:
This is going to be an internal credential type like the AzureCliCredential.

Reply:
Fixed